nav2_bt_navigator/CMakeLists.txt

```diff
@@ -22,16 +22,18 @@ include_directories(
   include
 )
 
-set(executable_name bt_navigator)
+set(bt_navigator_library_name bt_navigator)
+set(navigate_to_pose_executable_name bt_navigator)
 
-add_executable(${executable_name}
-  src/main.cpp
+add_executable(${navigate_to_pose_executable_name}
+  src/navigate_to_pose/main.cpp
 )
 
-set(library_name ${executable_name}_core)
+set(library_name ${bt_navigator_library_name}_core)
 
 add_library(${library_name} SHARED
   src/bt_navigator.cpp
+  src/navigate_to_pose/bt_navigator.cpp
```

Comment:
Are there two different files with the same name?

Reply:
- `nav2_bt_navigator/*/bt_navigator.cpp|.hpp`: Implementation of BtNavigator.
- `nav2_bt_navigator/*/navigate_to_pose/bt_navigator.cpp|.hpp`: Implementation of NavigateToPoseBtNavigator. NavigateToPoseBtNavigator inherits from BtNavigator.

I know it's a confusing name, so I'm thinking about it.